src/js/services/RecipeService.js

```diff
@@ -51,20 +51,29 @@ function parseRecipe(promise) {
 
 function RecipeService($http, $q, AuthService) {
     return {
-        getFeaturedRecipes: () => {
-            return parseRecipeArray($http.get(`http://api.kitchen.support/recipes/featured?api_token=${AuthService.getApiToken()}`));
+        getRecipeStream: () => {
+            return parseRecipeArray($http.get(`http://api.kitchen.support/stream`));
         },
         getSearch: (searchTerm) => {
-            return parseRecipeArray($http.get(`http://api.kitchen.support/recipes/search/${searchTerm}`));
+            return parseRecipeArray($http.get(`http://api.kitchen.support/recipes/search/${searchTerm}?forceNew=true`));
         },
         getRecipe: (recipeId) => {
-            return parseRecipe($http.get(`http://api.kitchen.support/recipes/recipe/${recipeId}`));
+            return parseRecipe($http.get(`http://api.kitchen.support/recipe?yummly_id=${recipeId}`));
         },
         favoriteRecipe: (recipeId) => {
-            return $http.post(`http://api.kitchen.support/recipes/favorite`, {api_token: AuthService.getApiToken(), id: recipeId});
+            return $http.post(`http://api.kitchen.support/recipes/favorite`, {api_token: AuthService.getApiToken(), recipe_id: recipeId});
         },
         unFavoriteRecipe: (recipeId) => {
-            return $http.post(`http://api.kitchen.support/recipes/unfavorite`, {api_token: AuthService.getApiToken(), id: recipeId});
+            return $http.delete(`http://api.kitchen.support/recipes/favorite`, {api_token: AuthService.getApiToken(), recipe_id: recipeId});
+        },
+        likeRecipe: (recipeId) => {
+            return $http.post(`http://api.kitchen.support/likes`, {api_token: AuthService.getApiToken(), recipe_id: recipeId});
+        },
+        unLikeRecipe: (recipeId) => {
+            return $http.delete(`http://api.kitchen.support/likes`, {api_token: AuthService.getApiToken(), recipe_id: recipeId});
```

Comment:
If we want to support both liking and disliking, will this state still be supported? As in, once someone explicitly likes or dislikes, will they be able to go back to neutral?